.github/workflows/deploy.yml

```
@@ -33,56 +36,88 @@ jobs:
+          --region $REGION \
+          --port 8080 \
+          --timeout 300s \
+          --cpu=1 \
+          --memory=512Mi \
+          --min-instances=1 \
+          --allow-unauthenticated
 
-    - name: Verify deployments
-      timeout-minutes: 10
+    - name: Verify Deployment
       run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:89: Double quote to prevent globbing and word splitting

(shellcheck)